core/runtime/pom.xml

```diff
@@ -43,6 +43,10 @@
             <groupId>io.smallrye</groupId>
             <artifactId>smallrye-config</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
```

Comment:
I can't say I'm excited to add a dependency to javax JSON in the core.I have another question: does it work for a native executable? I would have expected the `quarkus-jsonp` extension to be necessary.

If so, it makes the case of a specific extension IMHO.

Reply:
It seemed to work.  But yeah maybe a specific extension is warranted.

Reply:
For the record - `quarkus-jsonp` only registers `org.glassfish.json.JsonProviderImpl` for reflection so that `javax.json.spi.JsonProvider` could [instantiate the class](https://github.com/javaee/jsonp/blob/master/api/src/main/java/javax/json/spi/JsonProvider.java#L96-L97).